lib/ip_wrangler/db.rb

```diff
@@ -0,0 +1,85 @@
+module IpWrangler
+  class DB
+    def initialize(db_name, logger)
+      @db = Sequel.connect('sqlite://' + db_name)
+      @logger = logger
+    end
+
+    def select_nat_port(private_ip=nil, private_port=nil, protocol=nil)
+      params = {:private_ip => private_ip, :private_port => private_port, :protocol => protocol}.select { |key, value| value != nil }
```

Comment:
Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used.<br>Line is too long. [133/80]<br>Use the new Ruby 1.9 hash syntax.<br>Explicit non-nil checks are usually redundant.<br>Space inside { missing.<br>Space inside } missing.